pipelines/manual-release-pipeline.yml

```diff
@@ -779,6 +780,33 @@ jobs:
   plan:
     - get: every-minute
 
+- name: "Preview Terraform Changes"
+  serial: true
```

Comment:
I don't think it's serial. Surely it's standalone so that it can be run without needing to run the serial group